lib/presentation/localizations/app_localizations.dart

```diff
@@ -446,6 +446,57 @@ class AppLocalizations {
       name: 'Some items could not be copied to Shared space',
     );
   }
+
+  String get delete {
+    return Intl.message(
+      'Delete',
+      name: 'Delete',
+    );
+  }
+
+  String are_you_sure_you_want_to_delete_file(String documentName) {
+    return Intl.message(
+      'Are you sure you want to delete "$documentName"?',
+      name: 'are_you_sure_you_want_to_delete_file',
+      args: [documentName],
+    );
+  }
+
+  String are_you_sure_you_want_to_delete_files(int documentCount) {
+    return Intl.message(
+      'Are you sure you want to delete $documentCount items?',
+      name: 'are_you_sure_you_want_to_delete_files',
+      args: [documentCount],
+    );
+  }
+
+  String get the_file_is_successfully_deleted {
+    return Intl.message(
+      'The file is successfully deleted',
+      name: 'the_file_is_successfully_deleted',
+    );
+  }
```

Comment:
we should have a string for all files have been deleted no  ?

Reply:
yes, I've already defined it `some_items_are_successfully_deleted`